packages/expo-random/src/ExpoRandom.web.ts

```diff
@@ -2,6 +2,10 @@ export default {
   get name(): string {
     return 'ExpoRandom';
   },
+  getRandomBytes(length: number): Uint8Array {
+    const array = new Uint8Array(length);
+    return window.crypto.getRandomValues(array);
```

Comment:
```suggestion
    return (window.crypto ?? window.msCrypto).getRandomValues(array);
```
? https://developer.mozilla.org/en-US/docs/Web/API/Window/crypto#Syntax